standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HiveAlterHandler.java

```diff
@@ -130,6 +130,11 @@ public void alterTable(RawStore msdb, Warehouse wh, String catName, String dbnam
       throw new InvalidOperationException("Invalid column " + validate);
     }
 
+    // Validate bucketedColumns in new table
+    if (!MetaStoreServerUtils.validateBucketColumns(newt.getSd())) {
+      throw new InvalidOperationException("Bucket column doesn't match with any table columns");
```

Comment:
Useful to add an error log with the column name which is missing from table columns list.

Reply:
1. Converted return type to List<String>.
2. Added Log.error() along with column name.
3. Added column to exception also.